src/core/service.c

```diff
@@ -69,6 +69,7 @@ static const UnitActiveState state_translation_table[_SERVICE_STATE_MAX] = {
         [SERVICE_RELOAD]                     = UNIT_RELOADING,
         [SERVICE_RELOAD_SIGNAL]              = UNIT_RELOADING,
         [SERVICE_RELOAD_NOTIFY]              = UNIT_RELOADING,
+        [SERVICE_RELOAD_MOUNTING]            = UNIT_RELOADING,
```

Comment:
This should be mapped to refreshing. As discussed in one of the meetings, eventually we want extension refreshes to be independent from reloading. I did b8fa23059 specifically to accommodate this PR, too